package.json

```diff
@@ -14,6 +14,9 @@
     "start": "yarn storybook",
     "test": "jest",
     "test:watch": "jest --watch",
+    "lint": "tslint src/**/*.{ts,tsx} -t verbose",
+    "lint:fix": "tslint --fix src/**/*.{ts,tsx} -t verbose",
+    "ts:type-check": "tsc --noEmit -p tsconfig.json",
```

Comment:
Do we need to use the `-p tsconfig.json` flag? It seems to work fine without on my machine? We could also rename this script to `lint:types`, not sure if `ts:type-check` is accepted as standard in the TS community?

```
"lint:types": "tsc --noEmit",
```

Reply:
You're right, there's no need for `-p` in this instance. Also yeah i'm unsure what the standard is for the script name. Happy to go with `lint:types` 👍

Reply:
renamed the script and removed `-p tsconfig.json`